apps/framework-docs/src/pages/faqs.mdx

```diff
@@ -0,0 +1,11 @@
+# FAQs
+
+---
+
+### Why is Docker is not working after making changes to some fields?
+
+Terminate the Docker containers and restart Docker manually. If you have problems restarting run `npm run dev` twice.
+
+### Why has a test run crashed?
```

Comment:
This gives the impression that there's a moose test framework they can run, which we don't yet have. Is this referring to sending data to ingestion point?

Reply:
I have rephrased this question for more clarity in the the most recent commit. Let me know if it sounds better.